src/datePicker/Day.tsx

```diff
@@ -47,7 +58,8 @@ export const Day: React.FC<Props> = ({
     let date = getDayFormat(day, jalali);
     return (
       disabledDays.includes(date) ||
-      (disabledBeforeToday && dayjs(day).isBefore(dayjs().format(FORMAT_DATE)))
+      (disabledBeforeDate && dayjs(day).isBefore(disabledBeforeDate)) ||
+      (disabledAfterDate && dayjs(day).isAfter(disabledAfterDate))
```

Comment:
Hey, That's Great, But you know we should handle it in Jalali too, so I suggestion change this part to this: 

` (disabledBeforeDate && dayjs(dayjs(date)).isBefore(disabledBeforeDate)) ||
      (disabledAfterDate && dayjs(dayjs(date)).isAfter(disabledAfterDate))`

Reply:
Curious: wouldn't `dayjs(date)` and `dayjs(dayjs(date))` produce the same thing? If so I think the double wrapping is redundant. I'll experiment in the story to see!

Reply:
I experimented with this, and neither `dayjs(jdate)` nor `dayjs(dayjs(jdate))` seems to parse Jalali dates when I set Jalali to true and pass in jalali dates - it just treats them as Gregorian. Is the expected behavior that dates in disabledDays/selectedDays should work when Jalali? I can test against this functionality, but will probably want to add some new stories so this is clear!

Reply:
Oh, sorry, I didn't know that, thanks. 
I think we have a bug with isAfter,
If we use `date` instead of  'day object' when we are in Jalali,  
The days after the `disabledAfterDate` will be disabled, 
But when we use 'day object', it makes the `disabledAfterDate` date and the days after that disabled.Please have a  look at the pictures.

with the `dayjs(date).isAfter(disabledAfterDate))`  and it's currect: 
<img width="1311" alt="Screen Shot 1400-01-09 at 00 23 52" src="https://user-images.githubusercontent.com/18388480/112766041-3c9c5480-9025-11eb-808c-6a7c2166f21f.png">

with the `dayjs(day).isAfter(disabledAfterDate))` and it's wrong: 
<img width="1312" alt="Screen Shot 1400-01-09 at 00 23 16" src="https://user-images.githubusercontent.com/18388480/112766094-838a4a00-9025-11eb-8e8e-6d940ddece8f.png">



So I think we need change this part of date and range picker to with `date` not `day`: 

`      (disabledBeforeDate && dayjs(date).isBefore(disabledBeforeDate)) ||
      (disabledAfterDate && dayjs(date).isAfter(disabledAfterDate))`

Reply:
Thanks so much for testing this! I'm pretty unfamiliar with the Jalali calendar but want to make sure I'm getting it right. I've switched both Day components to use the post-format date which should address this!